tests/integration/s2n_test_constants.py

```diff
@@ -63,6 +63,23 @@
     S2N_CIPHER("DHE-RSA-CHACHA20-POLY1305", S2N_GNUTLS_PRIORITY_PREFIX + ":+DHE-RSA:+CHACHA20-POLY1305:+AEAD", S2N_TLS12, True, False),
 ]
 
+# Expected preferences for SignatureAlgorithms in GnuTLS priority string format
```

Comment:
Add reference to corresponding header file with sig alg prefs? May be helpful when updating in the future.

Reply:
good idea! will do